tests/test_features/test_pca.py

```diff
@@ -120,7 +120,7 @@ def test_discrete(self):
         """
         Test discrete target.
         """
-        colors = ["Y", "C0"]
+        colors = ["y", "C0"]
```

Comment:
This was issuing a deprecation warning that upper case single letter colors are no longer allowed.